### ISSUE TITLE
name_prefix parameter does not prefix string on resources names 

### ISSUE THREAD

**Issue by mailuminatti** [superblocksteam/terraform-aws-superblocks#4]
The variable `name_prefix` is intended to prefix a fixed string on resources names created by the terrform module `terraform_aws_superblocks`, however, nothing is appended in the resources names.
Please note that as some services are global, this makes this module not able to be used in multiple accounts by prefixing `staging` or `production` for example (as the name of global resources have to be unique through all the AWS accounts within an organization)

Example:
```terraform

provider "aws" {
  region = "us-east-1"
}

[...]

  module "terraform_aws_superblocks" {
    source  = "superblocksteam/superblocks/aws"
    version = ">=0.1.0"
  
    vpc_id         = [Redacted]
    lb_subnet_ids  = [Redacted]
    ecs_subnet_ids = [Redacted]
    domain         = "superblocks-staging.mydomain.com"
    subdomain      = "agent"

    superblocks_agent_environment = "staging"
    name_prefix = "staging"
  }
```

Partial `terraform plan` output (names highlighted) 

```
martin@server:~/git/superblocks/tf-superblocks-dev$ terraform plan

  # module.terraform_aws_superblocks.module.ecs[0].aws_ecs_service.superblocks will be created
  + resource "aws_ecs_service" "superblocks" {
      + cluster                            = (known after apply)
      + deployment_maximum_percent         = 200
      + deployment_minimum_healthy_percent = 100
      + desired_count                      = 1
      + enable_ecs_managed_tags            = false
      + enable_execute_command             = false
      + iam_role                           = (known after apply)
      + id                                 = (known after apply)
      + launch_type                        = "FARGATE"
      🛑+ name                               = "superblocks-service"
      + platform_version                   = (known after apply)
      + scheduling_strategy                = "REPLICA"
      + tags_all                           = (known after apply)
      + task_definition                    = "arn:aws:ecs:us-east-1:[REDACTED]:task-definition/superblocks-agent:8"
      + triggers                           = (known after apply)
      + wait_for_steady_state              = false

      + load_balancer {
         🛑+ container_name   = "superblocks-agent"
          + container_port   = 8020
          + target_group_arn = "arn:aws:elasticloadbalancing:us-east-1:[REDACTED]:targetgroup/superblocks-target-group/8e006338ab736a37"
        }

      + network_configuration {
          + assign_public_ip = false
          + security_groups  = [
              + [REDACTED],
            ]
          + subnets          = [
              + [REDACTED],
              + [REDACTED],
              + [REDACTED],
            ]
        }
    }
```
Expected result: 

Resources created through the use of the terraform module have the `staging` string appended

**Comment by edy-chan** [comment]
Vote up.

A pull request here: https://github.com/superblocksteam/terraform-aws-superblocks/pull/10
Waiting for code owners to review

**Comment by taha-au** [comment]
PR #10 has been merged, and v0.1.7 has been released with it
